### PR TITLE
Adding functionality to run multiple tasks via command line

### PR DESCRIPTION
**Pull request by carakessler**
https://github.com/checkupjs/checkup/issues/525

### CODE REVIEW

File: packages/cli/src/task-list.ts [hunk 1]
```diff
@@ -71,6 +71,36 @@ export default class TaskList {
     return this.getTasks().find((task) => task.meta.taskName === taskName);
   }
 
+  /**
+   * Finds tasks by task name
+   *
+   * @method findTasks
+   * @param taskNames The name of the task to find
+   */
+  findTasks(taskNames: TaskName[]): { tasksFound: Task[]; tasksNotFound: TaskName[] } {
```
Comment (scalvert) [inline packages/cli/src/task-list.ts:80]: Could we change this to use varargs instead? 

```suggestion
  findTasks(...taskNames: TaskName[]): { tasksFound: Task[]; tasksNotFound: TaskName[] } {
```

File: packages/cli/src/commands/run.ts [hunk 2]
```diff
@@ -126,26 +137,21 @@ export default class RunCommand extends BaseCommand {
   private async runTasks() {
     [this.metaTaskResults, this.metaTaskErrors] = await this.defaultTasks.runTasks();
 
-    if (this.runFlags.task !== undefined) {
-      let pluginTaskResult: TaskResult | undefined;
-      let taskFound: boolean = false;
+    if (this.runFlags.tasks !== undefined) {
+      let { tasksFound, tasksNotFound } = this.pluginTasks.findTasks(this.runFlags.tasks);
```
Comment (scalvert) [inline packages/cli/src/commands/run.ts:141]: If we change the signature of `findTasks` as per my suggestion below, I think you may need to spread this?

```suggestion
      let { tasksFound, tasksNotFound } = this.pluginTasks.findTasks(...this.runFlags.tasks);
```

File: packages/cli/src/reporters/console-reporter.ts [hunk 3]
```diff
@@ -180,6 +180,14 @@ let outputMap: { [taskName: string]: (taskResult: TaskResult) => void } = {
       );
     });
   },
+  foo: function (taskResult: TaskResult) {
```
Comment (scalvert) [inline packages/cli/src/reporters/console-reporter.ts:183]: Such foo...

Comment (carakessler) [reply packages/cli/src/reporters/console-reporter.ts:183]: lollllll

**Pull request merged**